spec/lib/rbac/filterer_spec.rb

```diff
@@ -35,7 +35,7 @@
         it "with :user finds #{klass}" do
           owned_resource = FactoryGirl.create(factory_name, :tenant => owner_tenant)
           _other_resource = FactoryGirl.create(factory_name, :tenant => other_tenant)
-          results = Rbac.filtered(klass, :user => owner_user)
+          results = described_class.filtered(klass, :user => owner_user)
```

Comment:
Aside: Random, good example of why I actually like `described_class` - If this spec had `described_class` to begin with, the only change in this file would have been line 1 to move all of this stuff over.